toolkit/tools/pkg/imagecustomizerlib/artifactsinputoutput.go

```diff
@@ -221,13 +221,8 @@ func InjectFiles(buildDir string, baseConfigPath string, inputImageFile string,
 	}
 	inputImageFormat := strings.TrimLeft(filepath.Ext(inputImageFile), ".")
 	rawImageFile := filepath.Join(buildDirAbs, BaseImageName)
-	imageInfo, err := getImageFileInfo(inputImageFile)
-	if err != nil {
-		return fmt.Errorf("failed to detect input image (%s) format:\n%w", inputImageFile, err)
-	}
-	outputImageFormat := imagecustomizerapi.ImageFormatType(imageInfo.Format)
 
-	err = convertImageToRaw(inputImageFile, inputImageFormat, rawImageFile)
+	outputImageFormat, err := convertImageToRaw(inputImageFile, inputImageFormat, rawImageFile)
```

Comment:
I could imagine there might be some users who will want to do format conversion after injection. e.g. converting to ISO or COSI. So, I am thinking it might be better to have the user specify the output format, instead of inferring it.